doc/source/io.rst

```diff
@@ -4668,6 +4668,36 @@ Passing ``index=True`` will *always* write the index, even if that's not the
 underlying engine's default behavior.
 
 
+Partitioning Parquet files
+''''''''''''''''''''''''''
+
+.. versionadded:: 0.24.0
+
+Parquet supports partitioning of data based on the values of one or more columns.
+
+.. ipython:: python
+   :suppress:
```

Comment:
don't add the suppress here (this won't print the block). Rather add another block after than cleans up the dir that is created (and use suppress on that)